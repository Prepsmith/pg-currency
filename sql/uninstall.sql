--
-- undefine the datatype(s).
--
SET client_min_messages = warning;
\set ECHO none
\i uninstall_currency.sql
RESET client_min_messages;
\set ECHO all

