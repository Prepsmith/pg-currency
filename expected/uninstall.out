--
-- undefine the datatype(s).
--
SET client_min_messages = warning;
SET
\set ECHO none
SET
DROP TYPE
DROP OPERATOR CLASS
DROP OPERATOR CLASS
DROP CAST
DROP CAST
DROP OPERATOR
DROP OPERATOR
DROP OPERATOR
DROP OPERATOR
DROP OPERATOR
DROP OPERATOR
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP FUNCTION
DROP TYPE
RESET
